src/ManageCourses.Api/Services/Publish/CourseMapper.cs

```diff
@@ -3,22 +3,24 @@
 using System.Collections.ObjectModel;
 using System.Linq;
 using System.Text;
-using GovUk.Education.ManageCourses.ApiClient.Helpers;
+using GovUk.Education.ManageCourses.Api.Model;
+using GovUk.Education.ManageCourses.Api.Services.Publish.Helpers;
+using GovUk.Education.ManageCourses.Domain.Models;
 using GovUk.Education.SearchAndCompare.Domain.Models;
 using GovUk.Education.SearchAndCompare.Domain.Models.Enums;
 using GovUk.Education.SearchAndCompare.Domain.Models.Joins;
+using Course = GovUk.Education.ManageCourses.Api.Model.Course;
 
 
-namespace GovUk.Education.ManageCourses.ApiClient
+namespace GovUk.Education.ManageCourses.Api.Services.Publish
```

Comment:
Wrong namespace?
Its just a course mapper that is used in both publishing and also the preview right?

Reply:
Moved to GovUk.Education.ManageCourses.Api.Mapping